services/user.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str | None = None,
+        first_name: str = "",
+        last_name: str = "",
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(pk=user_id)
```

Comment:
@yashkunn Regarding your question why pk only here
it's always preferable to use pk over id because in the future you may change the primary key (for example name that is unique instead of id). In this case everything will break cuz you hardcoded id field instead of dynamic pk. Oleksandr mentioned it once so that you know about this issue and can fix it everywhere ;)